docker/start-bmlt.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
-set -o xtrace
 set -e
 
-/bin/bash /tmp/write-config.sh
+echo "Waiting for database..."
+until /usr/bin/php /var/www/html/main_server/artisan db:show >/dev/null 2>&1; do
+  sleep .2
+done
+echo "connected"
+
+/usr/bin/php /var/www/html/main_server/artisan migrate --force
+/usr/bin/php /var/www/html/main_server/artisan settings:sync --force
```

Comment:
If the migrations are already syncing the settings, can this line be removed?

Reply:
no because the `settings:sync` command will read from environment variables, as we are removing the `[write-config.sh](https://github.com/bmlt-enabled/bmlt-server/blob/main/docker/write-config.sh)` we need this because of that too